client/signup/accordion-form/form-components.tsx

```diff
@@ -90,10 +93,24 @@ interface TextInputFieldProps {
 	onChange?: ( event: ChangeEvent< HTMLInputElement > ) => void;
 }
 
+export function TextInputLabel( {
+	inputName,
+	labelText,
+}: {
+	inputName?: string;
+	labelText?: TranslateResult;
+} ) {
+	return (
+		<LabelContainer>
+			<Label htmlFor={ inputName }>{ labelText }</Label>
```

Comment:
Nit: I was wondering if this needs to be wrapped in `LabelContainer`? Maybe we can add the margin to the `Label` styled component?

Instead of accepting the `labelText` prop, can we instead use `props.children` in the output? If yes, then we probably don't need the `TextInputLabel` component and we can directly use the `Label` component instead. 🤔

Reply:
Actually I wanted to wrap the label with a semantically valid block level element(div) so that block level formatting can be done. This is the reason to wrap the label with a label container.